spring-cloud-previews/google-cloud-apigee-connect-spring-starter/src/main/java/com/google/cloud/apigeeconnect/v1/spring/TetherSpringProperties.java

```diff
@@ -38,8 +38,6 @@ public class TetherSpringProperties implements CredentialsSupplier {
   private String quotaProjectId;
   /** Number of threads used for executors. */
   private Integer executorThreadCount;
-  /** Allow override of default transport channel provider to use REST instead of gRPC. */
```

Comment:
Removal of unused/unsupported property (this library has no rest-supported rpc methods)